### PR TITLE
v33

### PR DESCRIPTION
**Pull request by adam-saland**

### CODE REVIEW

**Comment by adam-saland** [comment]
fix linting

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
           npm install
       - name: Run prettier
         run: |-
-          npx prettier --config .prettierrc --check .
+          npm run -- prettier --config .prettierrc --check .
```
Comment (obany) [inline .github/workflows/lint.yml:18]: Why does this need changing, it works in the ws repo ?

Comment (adam-saland) [reply .github/workflows/lint.yml:18]: Idk, npx is not a command

Comment (adam-saland) [reply .github/workflows/lint.yml:18]: nvm 1 sec.

**Review by obany — APPROVED** [review]

**Pull request merged**